File: backend/cebunest/src/main/java/edu/cit/saligue/cebunest/dto/RegisterRequest.java
package edu.cit.saligue.cebunest.dto;

import lombok.Data;

@Data
public class RegisterRequest {
    private String name;
    private String email;
    private String phoneNumber;
    private String password;
    private String confirmPassword;
    private String role;
    private String facebookUrl;
    private String instagramUrl;
    private String twitterUrl;
}

File: mobile/app/src/main/java/com/cebunest/app/model/RegisterRequest.kt
package com.cebunest.app.model

import com.google.gson.annotations.SerializedName

data class RegisterRequest(
    val name: String,
    val email: String,
    val password: String,
    @SerializedName("confirmPassword") val confirmPassword: String,
    @SerializedName("phoneNumber") val phoneNumber: String = "",
    val role: String = "TENANT"
)